PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/navbar/navbar.component.html

```diff
@@ -2,32 +2,38 @@
     <div id="navbar-item-container">
         <pokeone-navbar-item 
             menuItemCaption="Locations"
+            menuItemLink="/l"
             menuItemImage="./assets/img/navbar/locations.png"
             [subMenuItems]="['Kanto', 'Johto', 'Unova', 'Sevii Islands']">
         </pokeone-navbar-item>
 
         <pokeone-navbar-item 
             menuItemCaption="Pokédex"
+            menuItemLink="/p"
             menuItemImage="./assets/img/navbar/pokedex.png">
         </pokeone-navbar-item>
 
         <pokeone-navbar-item 
             menuItemCaption="Items"
+            menuItemLink="/i"
             menuItemImage="./assets/img/navbar/items.png">
         </pokeone-navbar-item>
 
         <pokeone-navbar-item 
-            menuItemCaption="Moves"
-            menuItemImage="./assets/img/navbar/moves.png">
+            menuItemCaption="Guides"
+            menuItemLink="/g"
+            menuItemImage="./assets/img/navbar/guides.png">
```

Comment:
Image is blurry. Use a pixel art scaling tool to scale the image up to 120x120px, i.e. https://lospec.com/pixel-art-scaler/

Reply:
Done